Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,21 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+var caps = ["A", "B", "C", "D", "E", "F", "G", "H", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
```

Comment:
This function works, well done.
Is there a way to simplify it? You could think about what the function needs to do, in two steps:
1. convert the whole `txt` string to upper case
2. replace any spaces with `_`

There's a string method called `toUpperCase()`, can you use it to do the case conversion in one line?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase

Reply:
I made it more simple and used toUpperCase() too. thanks